src/fragrance_rater/api/imports.py

```
@@ -0,0 +1,68 @@
+            await session.commit()
+
+        return ImportResponse(
+            total_rows=result.total_rows,
+            imported=result.imported,
+            skipped=result.skipped,
+            errors=result.errors[:20],  # Limit errors returned
+        )
+
+    finally:
+        # Clean up temp file
+        tmp_path.unlink(missing_ok=True)
```

Ruff (0.15.12)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)